lib/resources.rb

```diff
@@ -108,13 +108,13 @@ def get_resource(key)
         end
       end
 
-      def authorization_types(practitioner: true, patient: false, system: false)
+      def authorization_types(provider: true, patient: false, system: false)
```

Comment:
let's default provider to false if we're passing it in anyways on each method call

either that or actually take advantage of the default

Reply:
defaulting to false https://github.com/cerner/fhir.cerner.com/pull/430/commits/56870f0ff4573ad640d54226948788ec51129ab0#diff-7718c11ed130ac00a9854e0b089f74b4R111

Reply:
need to fix up some specs

Reply:
let's add an .rspec file as well

Reply:
https://github.com/cerner/fhir.cerner.com/pull/430/commits/204ced2476ff61648bb6e80f9cebee4c95f0aeab